frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,68 @@
+import { FormEvent, useContext, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { AuthContext } from "../contexts/AuthContext";
+
+import styles from "./Login.module.css";
+
+export function Login() {
+  const { isLoggedIn, login } = useContext(AuthContext);
+  const navigate = useNavigate();
+
+  // Redirect once logged in
+  useEffect(() => {
+    if (isLoggedIn) navigate("/apply");
+  }, [isLoggedIn]);
+
+  function handleSubmit(event: FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    const target = event?.target as HTMLFormElement | null;
+
+    if (!target || target?.length < 2) {
+      alert("Error submitting login!");
```

Comment:
As opposed to alert it might be better to display the text on the page instead of using Alerts

Reply:
Can we postpone this feature until when we redo the login page? I'm not exactly sure what the designs would be for these error messages/ what the final login page design will be.